api/src/test/java/org/openmrs/migration/MigrationHelperTest.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.migration;
+
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Created by freddy on 20.05.18.
+ */
+public class MigrationHelperTest {
```

Comment:
@fruether thanks for bringing this to my attention. the `MigrationHelper` is never used within openmrs, I also searched the openmrs organisation on github and no usage exists. I will create an issue to deprecate/remove it from openmrs as it was introduced in 2006 and all the commits I see related to this file afterwards are just to keep it from breaking our build. Please remove this test class. Your work on this file is not for nothing, it did help us clean up our code! Thanks a lot!!